packages/runtime/container-runtime/src/gc/garbageCollection.ts

```diff
@@ -581,22 +595,16 @@ export class GarbageCollector implements IGarbageCollector {
 		);
 
 		// 4. Run the Sweep phase.
-		// It will delete sweep ready nodes and return a list of deleted node ids.
-		const deletedNodeIds = this.runSweepPhase(
-			gcResult,
-			tombstoneReadyNodeIds,
-			sweepReadyNodeIds,
-		);
+		// It will initiate the deletion of sweep ready nodes by sending a sweep op. All clients, including this one, will
+		// delete these nodes once it processes the op.
```

Comment:
I forgot to update this to mention that Tombstone happens too.  How about this?

```suggestion
		// 4. Run the Sweep phase.
		// It will Tombstone any tombstone-ready nodes, and initiate the deletion of sweep-ready nodes by sending a sweep op.
		// All clients, including this one, will delete these nodes once it processes the op.
```

Reply:
Ok and also the comment for Mark Phase should mention tombstone.  Oops.